eventlog/el/main.go

```diff
@@ -347,3 +411,327 @@ func search(c *cli.Context) error {
 	}
 	return nil
 }
+
+func login(c *cli.Context) error {
```

Comment:
is this the part that will be moved?

Reply:
Honestly, my plan was to leave it until we had another Go caller want to do the same, and then use the opportunity of two callers to clean it up into a shared library. It is a bit hard to imagine how different callers will want to do it; login is an interactive process.